### PR TITLE
Fixes the doc build

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -67,7 +67,7 @@ matrix:
          - python: 3.6
            env: SETUP_CMD='build_docs -w'
                 EVENT_TYPE='push pull_request cron'
-                SPHINX_VERSION='<1.6'
+                SPHINX_VERSION='>=1.6'
```
Comment (Cadair) [inline .travis.yml]: drop this completely.

File: docs/conf.py [hunk 2]
```diff
@@ -67,7 +67,7 @@
 # -- General configuration ----------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-needs_sphinx = '1.5'
+needs_sphinx = '1.6'
```
Comment (Cadair) [inline docs/conf.py]: are we requiring 1.6?

File: .travis.yml (hunk 1 above)
Comment (Cadair) [inline .travis.yml]: Is ci-helpers still pinning to 1.5?

File: docs/conf.py (hunk 2 above)
Comment (nabobalis) [inline docs/conf.py]: BOOO

File: .travis.yml (hunk 1 above)
Comment (nabobalis) [inline .travis.yml]: BOOO

File: docs/conf.py (hunk 2 above)
Comment (nabobalis) [inline docs/conf.py]: WHY DO YOU STOP PROGRESS?

**Comment by Cadair** [comment]
can we open an issue for those skipped tests.

**Pull request merged**